.github/workflows/auto-recode.yml

```diff
@@ -0,0 +1,60 @@
+name: auto-recoder
```

Comment:
Does it mean re-code? I think this name does not represent what this workflow does.

Reply:
I changed name auto-recoder to pr-number-assign!

Reply:
The file name remains the same, but I think it would be good to change the file name.